db/schema.rb

```diff
@@ -16,7 +16,7 @@
   enable_extension "plpgsql"
   enable_extension "pg_stat_statements"
 
-  create_table "api_keys", id: :serial, default: %q{nextval('api_keys_id_seq'::regclass)}, force: :cascade do |t|
```

Comment:
do we know what this nextval thing was and what it meant?

Reply:
I think that was an older versions way of doing it? I'm having trouble digging up info on that, I can say that rails5 definitely generates that as the default ```not null | nextval('projects_id_seq'::regclass)```

Reply:
Meant to copy/paste the api_keys version ```nextval('projects_id_seq'::regclass)```